pre-commit

```diff
@@ -22,12 +22,11 @@ for i in $(git diff --name-only --cached); do
     filename=$(basename -- "$i")
     extension="${filename##*.}"
     if [ "$extension" = "rs" ]; then
-        # Read rustfmt config, replace '\n' with ','
-        rustfmt_config="$(sed -z "s/\n/,/g;s/,$/\n/" ./tests/fmt.toml)"
         # We first do a check run, this will fail when it finds a non-matching license.
-        rustfmt $i --check --config $rustfmt_config
-        # Run `cargo fmt` for this file
-        rustfmt $i --config $rustfmt_config
+        rustfmt $i --check --config-path ./tests/fmt.toml
+
+        # Run `cargo fmt` for this source file using the fmt.toml config
+        rustfmt $i --config-path ./tests/fmt.toml
```

Comment:
The primary reason I used this approach was essentially to circumvent needing to include nightly. If you are including nightly it might be best to simply use a `rustfmt.toml` file (see https://rust-lang.github.io/rustfmt/?version=v1.5.1&search=).